dist/tools/codespell/ignored_words.txt

```diff
@@ -0,0 +1,17 @@
+# This file contains ignored words that triggered false positive with codespell
+#
+# Extend it when needed, one word per line with documentation in a comment.
+
+# nd (Neighbor Discovery) => and, 2nd
+ND
```

Comment:
here and below: for codespell, only the spelling of its own dictionary entry counts:

```
  -I FILE, --ignore-words FILE                                                                                                     
                        File that contains words which will be ignored by
                        codespell. File must contain 1 word per line. Words
                        are case sensitive based on how they are written in  
                        the dictionary file                                                                                        
```

Thus the capital spellings (ND, WAN) can be removed.

Reply:
Addressed.